flathub.json

```diff
@@ -0,0 +1,6 @@
+{
+    "only-arches": [
+        "x86_64"
+    ],
+    "publish-delay-hours": 0
```

Comment:
This should be removed. It's not an extra-data app. The publishing delay exists for a reason.

Reply:
btw this won't run on a aarch64 either which is a 64-bits computer. So let's not confuse architecture and width.

Reply:
This is strictly about `publish-delay-hours`.